src/Illuminate/Queue/Events/JobExceptionCaught.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Illuminate\Queue\Events;
+
+class JobExceptionCaught
+{
+    /**
+     * The connection name.
+     *
+     * @var string
+     */
+    public $connectionName;
+
+    /**
+     * The job instance.
+     *
+     * @var \Illuminate\Contracts\Jobs\Job
+     */
+    public $job;
+
+    /**
+     * The data given to the job.
+     *
+     * @var array
+     */
+    public $data;
+
+    /**
+     * The exception instance.
+     *
+     * @var \Exception|\Throwable
```

Comment:
Should I change all other occurences of \Exception|\Throwable to \Throwable only or only in here?